src/electionguard/serialize.py

```diff
@@ -48,13 +99,15 @@ def construct_path(
     return os.path.join(target_path, target_file)
 
 
-def from_raw(type_: Type[_T], obj: Any) -> _T:
-    """Deserialize raw as type."""
+def from_raw(type_: Type[_T], raw: Union[str, bytes]) -> _T:
+    """Deserialize raw json string as type."""
 
-    return parse_raw_as(type_, obj)
+    if type_ is datetime:
+        return parse_raw_as(type_, raw)
```

Comment:
This is tech debt. There are several tests that specifically work with datetime to test the deserialization but it's not placed in as a json format. This should become and issue to fix up those tests.